websites/E/EternalNetwork TM/presence.ts

```diff
@@ -158,17 +158,96 @@ presence.on("UpdateData", async () => {
 			if (pathname.startsWith("/")) presenceData.state = ttl;
 
 			if (pathname.includes("/admin")) {
-				presenceData.state = "Adding new incident 😥 !";
+				presenceData.state = "Adding new incident";
 				presenceData.smallImageText = "Admin Panel";
 				delete presenceData.buttons;
 			}
 
 			if (search.includes("?do=settings"))
-				presenceData.state = "Adding new service !";
+				presenceData.state = "Adding new service";
 			delete presenceData.buttons;
 
 			break;
 		}
+		case `et-log.${etrnl}`:
+		case `www.et-log.${etrnl}`: {
+			presenceData.smallImageKey = "eternallogistic_logo";
+
+			if (pathname.startsWith("/")) presenceData.state = ttl;
+
+			if (pathname.includes("/about.php")) {
+				presenceData.details = "Reading information";
+				presenceData.state = ttl;
+			}
+
+			if (pathname.includes("/gallery.php")) {
+				presenceData.details = "Checking gallery photos";
+				presenceData.state = ttl;
+			}
+
+			if (pathname.includes("/applications.php")) {
+				presenceData.details = "Applying for the VTC";
+				presenceData.state = ttl;
+			}
+
+			break;
+		}
+		case `dev.${etrnl}`:
+		case `www.dev.${etrnl}`: {
+			presenceData.smallImageKey = "eternalnetworktm_dev";
+
+			if (pathname.startsWith("/")) {
+				presenceData.state = "Creating new features";
+				presenceData.smallImageText = "Dev's paradise";
+				delete presenceData.buttons;
+			}
+
+			if (pathname.includes("/etlog")) {
+				presenceData.state = "Working on ET-LOG system";
+				presenceData.smallImageText = "Updating the website";
+				delete presenceData.buttons;
+			}
+
+			if (pathname.includes("/radio")) {
+				presenceData.state = "Working on radio page";
+				presenceData.smallImageText = "Updating the website";
+				delete presenceData.buttons;
+			}
+
+			break;
+		}
+		case `shortener.${etrnltm}`:
+		case `www.shortener.${etrnltm}`: {
+			presenceData.smallImageKey = "eternalshortener_logo";
+
+			if (pathname.startsWith("/")) presenceData.state = ttl;
+
+			if (pathname.includes("/home")) {
+				presenceData.state = ttl;
+				presenceData.smallImageText = "Adding new short url";
+				delete presenceData.buttons;
+			}
+
+			if (pathname.includes("/profile")) {
+				presenceData.state = "Checking profile";
```

Comment:
You could also include the profile name they user is viewing too
```suggestion
				presenceData.state = "Viewing profile";
```

Reply:
Fixed